tests/test_all.py

```diff
@@ -35,5 +50,7 @@ def test_all(queries, presto_op):
         if len(adjusted_opts) > 0:
             test_ops['test-query-' + str(ind + 1) + '.txt'] = adjusted_opts
 
+    print(test_ops)
```

Comment:
Do we want to keep this?

Reply:
I think its useful to see the output sometimes, but I can take it out?